Cargo.toml

```diff
@@ -94,6 +97,7 @@ progenitor = { git = "https://github.com/oxidecomputer/progenitor", branch = "ma
 propolis = { path = "lib/propolis", default-features = false }
 propolis-client = { path = "lib/propolis-client" }
 propolis-server-config = { path = "crates/propolis-server-config" }
+propolis-standalone-config = { path = "crates/propolis-standalone-config" }
```

Comment:
It'd be nice to be able to just consolidate these and have a single config. The on-disk format doesn't necessarily have to change but maybe something like `Config::new_standalone`/`Config::new_server`? Or even just a simpler first step would be keeping both in a common crate `StandaloneConfig`/`ServerConfig`.

Reply:
I've kept the standalone config stuff separate since we seem more likely to do weird stuff there than in propolis-server.  I'd really like to keep the option of experimentation there without having to worry about breaking something in -server.